selfdrive/car/hyundai/values.py

```diff
@@ -20,7 +20,8 @@ class CAR:
   GENESIS_G80 = "GENESIS G80 2017"
   GENESIS_G90 = "GENESIS G90 2017"
   HYUNDAI_GENESIS = "HYUNDAI GENESIS 2015-2016"
-  IONIQ = "HYUNDAI IONIQ ELECTRIC PREMIUM SE 2020"
+  IONIQ = "HYUNDAI IONIQ HYBRID"
+  IONIQ_EV_2020 = "HYUNDAI IONIQ ELECTRIC PREMIUM SE 2020"
```

Comment:
Let's just call it `IONIQ_EV` since it looks like both 2020 Ioniq models have everything standard

Reply:
Ok no problem